jb-lib/lib/file_storage/gcp/gcp_storage.py

```diff
@@ -0,0 +1,66 @@
+import os
+from typing import Union, Optional
+from dotenv import load_dotenv
+import logging
+from gcloud.aio.storage import Storage, Blob
+from ..storage import AsyncStorage
+
+load_dotenv()
+
+logger = logging.getLogger("storage")
+class GCPAsyncStorage(AsyncStorage):
+    __client__ = None
+    tmp_folder = "/tmp/jb_files"
+
+    def __init__(self):
+        logger.info("Initializing GCP Storage")
+        bucket_name = os.getenv("GCP_BUCKET_NAME")
+        if not bucket_name:
+            raise ValueError(
+                "GCPAsyncStorage client not initialized. Missing bucket_name "
+            )
+        self.__bucket_name__ = bucket_name
+        self.__client__ = Storage()
```

Comment:
Please check this Storage() instance from google package when u try to run this, it throws an error. Also try all the functions with keys to confirm whether the packages are working or not.